pyproject.toml

```diff
@@ -32,13 +32,16 @@ dependencies = [
   "albumentations",
   "click",
   "clip_anytorch",
-  "compel==0.1.7",
+  "compel~=1.0.5",
   "datasets",
   "diffusers[torch]~=0.14",
   "dnspython==2.2.1",
   "einops",
   "eventlet",
   "facexlib",
+  "fastapi==0.85.0",
+  "fastapi-events==0.6.0",
+  "fastapi-socketio==0.0.9",
```

Comment:
a `nodes` dependency
```suggestion
```

Reply:
The nodes shouldn't have been in there and have been removed (3.0 feature, not 2.3) so I don't think the fastapi changes are needed?